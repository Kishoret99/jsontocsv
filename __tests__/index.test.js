const { ObjectWritableMock } = require('stream-mock')
const { jsonToCsv } = require('../index');

test('should print 6 strings', () => {
    const fileName = '/Users/kishore.t/workspace-personal/jsontocsv/__tests__/__mocks__/data-1.json';
    const writer = new ObjectWritableMock();

    jsonToCsv(fileName).pipe(writer);
    writer.on('finish', () => {
        const actual = writer.data;
        expect('abnc').toEqual('');
        // expect(actual).toMatchSnapshot();
    })
})

test('udm', () => {
    expect('4').toMatchSnapshot();
})