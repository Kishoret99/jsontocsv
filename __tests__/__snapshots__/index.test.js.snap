// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`udm 1`] = `"3"`;
